BenchmarkVCFs/BenchmarkVCFs.wdl

```diff
@@ -20,15 +21,14 @@ workflow Benchmark {
         Array[String]? variantSelectorLabels
         String referenceVersion
         Int? threadsVcfEval=2
+        Int? threadsHappy=4
```

Comment:
Workflow currently doesn't run happy or GenotypeConcordance, only vcfeval.  If we want to include happy we should add the call to the task before adding this parameter.  Personally I would prefer to avoid adding happy, as it's much slower and doesn't seem to be any better than vcfeval.